### PR TITLE
Release 0.1.1

### PR DESCRIPTION
**Pull request by ZainPatelQB**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

## Motivation and Context
Why was this PR created?

## How has this been tested?
What testing strategies have you used?

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Assigned myself to the PR
- [ ] Added tests to cover my changes
- [ ] Added new entries to the `RELEASE.md` file

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Notice
 
-- [ ] I acknowledge and agree that, by checking this box and clicking “Submit Pull Request”:
+- [ ] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":
```
Comment (yetudada) [inline .github/PULL_REQUEST_TEMPLATE.md]: We might as well make some of the outstanding changes to Kedro-Docker now, can this notice be moved to the bottom of the PR template? Under checklist.

File: legal_header.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2018-2019 QuantumBlack Visual Analytics Limited
```
Comment (yetudada) [inline legal_header.txt]:
```suggestion
# Copyright 2020 QuantumBlack Visual Analytics Limited
```

File: tools/license_and_headers.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+import glob
+
+PATHS_REQUIRING_HEADER = ["kedro_docker", "tests"]
+LEGAL_HEADER_FILE = "legal_header.txt"
+LICENSE_MD = "LICENSE.md"
+
+RED_COLOR = "\033[0;31m"
+NO_COLOR = "\033[0m"
+
+LICENSE = """Copyright 2018-2019 QuantumBlack Visual Analytics Limited
```
Comment (yetudada) [inline tools/license_and_headers.py]:
```suggestion
LICENSE = """Copyright 2020 QuantumBlack Visual Analytics Limited
```

File: legal_header.txt (hunk 2 above)
Comment (yetudada) [reply legal_header.txt]: Because I've updated it here, it will need to be updated across the files it's on.

**Comment by yetudada** [comment]
And I can't remember, what was the status on Appveyor?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ZainPatelQB) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done in this separate PR https://github.com/quantumblacklabs/kedro-docker/pull/18

**Pull request merged**